src/main/java/com/sap/sgs/phosphor/fosstars/nvd/NVD.java

```diff
@@ -135,14 +186,11 @@ String location() {
         }
 
         for (NvdEntry entry : MAPPER.readValue(parser, NvdEntry[].class)) {
-          for (Matcher matcher : MATCHERS) {
-            if (matcher.match(entry, vendor, product)) {
-              nvdEntries.add(entry);
-            }
-          }
+          nvdEntries.put(entry.getCve().getCveDataMeta().getID(), entry);
```

Comment:
What if getCveDataMeta() is null?

Reply:
That would mean that an entry in NVD doesn't have a CVE id. Currently, NVD doesn't have such entries. I can add null-checks for completeness.